pkg/azure/resources.go

```diff
@@ -33,8 +33,17 @@ import (
 )
 
 const (
-	virtualMachineTypeName = "Microsoft.Compute/virtualMachines"
-	managedClusterTypeName = "Microsoft.ContainerService/managedClusters"
+	virtualMachineTypeName        = "Microsoft.Compute/virtualMachines"
```

Comment:
Curious if these constants was supposed to be only for supported compute resource types or if it makes sense to extend it to all the resource types that the plugin manages. I understand that these aren't exported so they are just an implementation detail, but I take all the existing plugins a sample code for people that want to implement more plugins, so I am curious if for these we are following similar pattern across plugins.

Reply:
> these constants was supposed to be only for supported compute resource types

Yea I think that was the intent but I couldn't find a better home for the rest of them. Open to ideas or leaving it for the time being. This doesn't really exist in other plugins as far as i know.